mnist/mnist_vanilla_k8s.ipynb

```diff
@@ -0,0 +1,909 @@
+{
```

Comment:
<p>Why do you have input statements? Couldn't the user just set these python variables?</p><p> </p><p>How about moving DOCKER_REGISTRY further up in the notebook? Ideally a user would just have to set one or two values in the first cell and then click run all to have the notebook run</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/examples/pull/737/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Changed it out to get this from the k8s python client</p>

Reply:
<p>Moved DOCKER_REGISTRY to the beginning of the notebook and added instructions to change it out</p>